app/src/main/java/org/schabi/newpipe/player/BackgroundPlayer.java

```diff
@@ -211,10 +218,7 @@ private void resetNotification() {
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
             setLockScreenThumbnail(builder);
         }
-
-        if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.JELLY_BEAN) {
-            builder.setPriority(NotificationCompat.PRIORITY_MAX);
-        }
+        builder.setPriority(NotificationCompat.PRIORITY_MAX);
```

Comment:
CC @friendlyanon

Reply:
This function seems to be available in every Android version, there are no notes about "JellyBean+ only" here: https://developer.android.com/reference/android/support/v4/app/NotificationCompat.Builder.html#setPriority(int)

Reply:
Please revert this change, as it has nothing to do with the PR and it doesn't fix nor improve anything: as far as I understand this was done automatically by the editor, so it's not something you decided to fix yourself